gen/topUtils.ml

```diff
@@ -54,9 +54,10 @@ module Make : functor (O:Config) -> functor (C:ArchRun.S) ->
     open Printf
     open Code
 
-    let pp_v =
-      if O.hexa then sprintf "0x%x"
-      else sprintf "%i"
+    let pp_v v =
+      let v = Code.value_to_int v in
+      if O.hexa then sprintf "0x%x" v
+      else sprintf "%i" v
```

Comment:
Hi @ShaleXIONG. Would it be meaningfull to print `NoValue` as, say `**` instead of  `-1` ?

Reply:
That is a good idea ! I will do it.